### PR TITLE
portuguese localization added

### PR DESCRIPTION
**Pull request by ramonfrombr**

### CODE REVIEW

**Comment by dmalan** [comment]
Hi @ramonfrombr, sorry for missing this. Did you indeed want to close?

File: submit50/locale/pt/LC_MESSAGES/submit50.po [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.8.0\n"
+
+#: submit50/__main__.py:64
+msgid ""
+"You have an unknown version of submit50. Please visit our status page "
+"https://cs50.statuspage.io for more information."
+msgstr ""
+"Você possui uma versão desconhecida do submit50. Por favor visite nossa página de status "
+"https://cs50.statuspage.io para mais informação."
```
Comment (felipap) [inline submit50/locale/pt/LC_MESSAGES/submit50.po:27]: "para mais informações"

File: submit50/locale/pt/LC_MESSAGES/submit50.po [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+#: submit50/__main__.py:122
+msgid "Files that won't be submitted:"
+msgstr "Arquivos que não serão enviados:"
+
+#: submit50/__main__.py:134
+msgid ""
+"Keeping in mind the course's policy on academic honesty, are you sure you"
+" want to submit these files (yes/no)? "
+msgstr ""
+"Tenha em mente a política de honestidade acadêmica do curso, você tem certeza"
```
Comment (felipap) [inline submit50/locale/pt/LC_MESSAGES/submit50.po:50]: "Tendo em mente"

File: submit50/locale/pt/LC_MESSAGES/submit50.po [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+"que quer enviar estes arquivos? (sim/não)"
```
Comment (felipap) [inline submit50/locale/pt/LC_MESSAGES/submit50.po:51]: @dmalan — should the options also be translated?

File: submit50/locale/pt/LC_MESSAGES/submit50.po [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+#: submit50/__main__.py:162
+msgid ""
+"Sorry, something's wrong, please try again. If the problem persists, "
+"please visit our status page https://cs50.statuspage.io for more "
+"information."
+msgstr ""
+"Desculpa, aconteceu algo de errado, por favor tente novamente. Se o problema persistir, "
+"por favor visite nossa página de status https://cs50.statuspage.io para mais"
+"informação"
```
Comment (felipap) [inline submit50/locale/pt/LC_MESSAGES/submit50.po:61]: Would also pluralize this here to "informações".

File: submit50/locale/pt/LC_MESSAGES/submit50.po [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+"info: adds all commands run.\n"
+"debug: adds the output of all commands run."
+msgstr ""
+"warning: exibe avisos de como utilizar.\n"
+"info: adiciona todos os comandos executados.\n"
+"debug: adiciona a saída de dados de todos os comandos executados."
+
+#: submit50/__main__.py:208
+msgid "prescribed identifier of work to submit"
+msgstr "é preciso enviar o identificador do problema como descrito nas instruções"
```
Comment (felipap) [inline submit50/locale/pt/LC_MESSAGES/submit50.po:91]: Should probably be a nominal phrase? Eg. "identificador do problema como descrito nas instruções"

**Review by felipap — COMMENTED** [review]
Few comments on the translations, otherwise looks good!

**Comment by ramonfrombr** [comment]
@dmalan I forgot I had opened this issue and didn't know that deleting the fork would impact it. I'll open it up again.

> Hi @ramonfrombr, sorry for missing this. Did you indeed want to close?

**Pull request closed without merging**